Scripts/Modules/TooltipModule/TooltipModule.cs

```diff
@@ -177,14 +188,27 @@ void UpdateVisuals(ITooltip tooltip, TooltipUI tooltipUI, Transform target, floa
 
 		public void OnRayEntered(GameObject gameObject, RayEventData eventData)
 		{
+			if (gameObject == this.gameObject)
+				return;
+
+			var tooltip = gameObject.GetComponent<ITooltip>();
+			if (tooltip != null)
+				ShowTooltip(tooltip);
+		}
+
+		public void OnRayHovering(GameObject gameObject, RayEventData eventData)
+		{
+			if (gameObject == this.gameObject)
+				return;
+
 			var tooltip = gameObject.GetComponent<ITooltip>();
 			if (tooltip != null)
```

Comment:
This came up for the "Main Menu is Hidden" tooltip on the MainMenuActivator. Because its state can change from no-tooltip to valid tooltip while you are hovering it, we need to check that. In this case, we have a null tooltip string up until the menu is hidden.  Same thing applies to checking while it's being shown whether the tooltip is still valid. If the menu moves away from a workspace while you're hovering the activator, the tooltip should hide, even though the ray is still hovering it.